libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -140,100 +138,95 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @Values{Copter, Rover, Plane}: 58:Clear Waypoints
     // @Values{Rover}: 59:Simple Mode
-    // @Values{Copter}: 60:ZigZag
+    // @Values{Copter}: 60:ZigZag Mode
     // @Values{Copter}: 61:ZigZag SaveWP
     // @Values{Copter, Rover, Plane}: 62:Compass Learn
     // @Values{Rover}: 63:Sailboat Tack
     // @Values{Plane}: 64:Reverse Throttle
     // @Values{Copter, Rover, Plane, Blimp}: 65:GPS Disable
     // @Values{Copter, Rover, Plane}: 66:Relay5 On/Off, 67:Relay6 On/Off
-    // @Values{Copter}: 68:Stabilize
-    // @Values{Copter}: 69:PosHold
-    // @Values{Copter}: 70:AltHold
-    // @Values{Copter}: 71:FlowHold
-    // @Values{Copter}: 72:Circle
-    // @Values{Plane}: 72:ModeCircle
-    // @Values{Copter}: 73:Drift
+    // @Values{Copter}: 68:STABILIZE Mode
+    // @Values{Copter}: 69:POSHOLD Mode
+    // @Values{Copter}: 70:ALTHOLD Mode
+    // @Values{Copter}: 71:FLOWHOLD Mode
+    // @Values{Copter,Plane}: 72:CIRCLE  Mode
+    // @Values{Copter}: 73:DRIFT Mode
     // @Values{Rover}: 74:Sailboat motoring 3pos
     // @Values{Copter}: 75:SurfaceTrackingUpDown
-    // @Values{Copter}: 76:Standby Mode
-    // @Values{Plane}: 77:ModeTakeoff
+    // @Values{Copter}: 76:STANDBY Mode
+    // @Values{Plane}: 77:TAKEOFF Mode
     // @Values{Copter, Rover, Plane}: 78:RunCam Control
     // @Values{Copter, Rover, Plane}: 79:RunCam OSD Control
     // @Values{Copter}: 80:VisOdom Align
-    // @Values{Rover}: 80:Viso Align
+    // @Values{Rover}: 80:VisoOdom Align
     // @Values{Copter, Rover, Plane, Blimp}: 81:Disarm
     // @Values{Plane}: 82:QAssist 3pos
     // @Values{Copter}: 83:ZigZag Auto
-    // @Values{Copter, Plane}: 84:Air Mode
+    // @Values{Copter, Plane}: 84:AirMode
     // @Values{Copter, Plane}: 85:Generator
-    // @Values{Plane}: 86: Non Auto Terrain Follow Disable
+    // @Values{Plane}: 86:Non Auto Terrain Follow Disable
     // @Values{Plane}: 87:Crow Select
     // @Values{Plane}: 88:Soaring Enable
     // @Values{Plane}: 89:Landing Flare
     // @Values{Copter, Rover, Plane, Blimp}: 90:EKF Pos Source
     // @Values{Plane}: 91:Airspeed Ratio Calibration
-    // @Values{Plane}: 92:FBWA
+    // @Values{Plane}: 92:FBWA Mode
     // @Values{Copter, Rover, Plane}: 94:VTX Power
     // @Values{Plane}: 95:FBWA taildragger takeoff mode
-    // @Values{Plane}: 96:trigger re-reading of mode switch
+    // @Values{Plane}: 96:Trigger re-reading of mode switch
     // @Values{Rover}: 97:Windvane home heading direction offset
-    // @Values{Plane}: 98: ModeTraining
+    // @Values{Plane}: 98:TRAINING Mode
     // @Values{Copter}: 99:AUTO RTL
     // @Values{Copter, Rover, Plane, Blimp}: 100:KillIMU1, 101:KillIMU2
     // @Values{Copter, Rover, Plane}: 102:Camera Mode Toggle
     // @Values{Copter, Rover, Plane}: 105:GPS Disable Yaw
     // @Values{Rover, Plane}: 106:Disable Airspeed Use
-    // @Values{Plane}: 107: EnableFixedWingAutotune
-    // @Values{Plane}: 108: ModeQRTL
+    // @Values{Plane}: 107:Enable FW Autotune
```

Comment:
Should this be "FW AutoTune Enable"?

Reply:
no,I think...FW AutoTune Enable....it really should be 'FW AutoTune Enable in other modes'....but that's too long